lndr-backend/src/Lndr/Db/Friendships.hs

```diff
@@ -25,3 +25,9 @@ lookupFriends addr conn =
 lookupFriendRequests :: Address -> Connection -> IO [UserInfo]
 lookupFriendRequests addr conn =
     query conn "SELECT inbound.origin, nicknames.nickname FROM friendships inbound LEFT JOIN friendships outbound ON inbound.friend = outbound.origin AND inbound.origin = outbound.friend LEFT JOIN nicknames ON nicknames.address = inbound.origin WHERE inbound.friend = ? AND outbound.friend IS NULL" (Only addr) :: IO [UserInfo]
+
+
+lookupPendingRequest :: Address -> Address -> Connection -> IO Int
+lookupPendingRequest addr friendAddr conn = do
```

Comment:
The name `lookupPendingRequest` is quite generic. A couple of key points:

 * Because of the uniqueness assertion on `(origin, friend)` on the friendships table, we know the return value here can only ever be `0` or `1` (i.e: boolean). Let's return a boolean, then.
 * This query is a filtered and reduced subset of the query above, `lookupFriendRequests`, so it can be similarly named.

Given these two points, I posit the name `hasFriendRequestFrom`.